src/docfx/config/ConfigLoader.cs

```diff
@@ -18,13 +19,41 @@ internal class ConfigLoader
         private readonly Input _input;
         private readonly RepositoryProvider _repositoryProvider;
 
+        /// <summary>
+        /// Since we currently do not support multiple docsets for localized repos without config,
+        /// this hack makes test happy.
+        /// </summary>
+        internal static Func<bool> DisableMultipleDocsets;
+
         public ConfigLoader(string docsetPath, Input input, RepositoryProvider repositoryProvider)
         {
             _docsetPath = docsetPath;
             _input = input;
             _repositoryProvider = repositoryProvider;
         }
 
+        public static (string docsetPath, string outputPath)[] FindDocsets(string workingDirectory, CommandLineOptions options)
+        {
+            if (DisableMultipleDocsets != null && DisableMultipleDocsets())
+            {
+                return new[] { (workingDirectory, options.Output) };
+            }
+
+            return Directory.GetFiles(workingDirectory, "docfx.yml", SearchOption.AllDirectories)
```

Comment:
Doesn't docfx v3 accept `docfx.json` as well?
Or does this mean multiple-docset should only run after config migration tool for Docs?

Reply:
yes, it should support `docfx.json`, see TODO comments in line 44